lib/formatters/checkstyle.js

```diff
@@ -38,15 +38,10 @@ module.exports = function(results) {
     results.forEach(function(result) {
         const messages = result.messages;
 
-        output += "<file name=\"" + xmlEscape(result.filePath) + "\">";
+        output += `<file name=\"${xmlEscape(result.filePath)}\">`;
 
         messages.forEach(function(message) {
-            output += "<error line=\"" + xmlEscape(message.line) + "\" " +
-                "column=\"" + xmlEscape(message.column) + "\" " +
-                "severity=\"" + xmlEscape(getMessageType(message)) + "\" " +
-                "message=\"" + xmlEscape(message.message) +
-                (message.ruleId ? " (" + message.ruleId + ")" : "") + "\" " +
-                "source=\"" + (message.ruleId ? xmlEscape("eslint.rules." + message.ruleId) : "") + "\" />";
+            output += `<error line=\"${xmlEscape(message.line)}\" column=\"${xmlEscape(message.column)}\" severity=\"${xmlEscape(getMessageType(message))}\" message=\"${xmlEscape(message.message)}${message.ruleId ? ` (${message.ruleId})` : ""}\" source=\"${message.ruleId ? xmlEscape(`eslint.rules.${message.ruleId}`) : ""}\" />`;
```

Comment:
I think maybe this one looks a bit more readable if we still split it into multiple lines. What do you think?